package.json

```diff
@@ -169,18 +163,13 @@
     "ts-essentials": "^5.0.0",
     "typescript": "~5.2.0",
     "urijs": "^1.18.12",
-    "webpack": "^5.96.1",
-    "webpack-cli": "^5.1.4",
-    "webpack-dev-server": "^5.1.0",
-    "worker-loader": "^3.0.8"
+    "webpack": "^5.96.1"
```

Comment:
I think it was svg-sprite-loader that tried to do webpack 2 (3?) version detection, and crashed if webpack couldn't be found at all.